text_machina/src/types.py

```diff
@@ -44,3 +44,10 @@ class TaskType(str, Enum):
     DETECTION: str = "detection"
     ATTRIBUTION: str = "attribution"
     BOUNDARY: str = "boundary"
+    MIXCASE: str = "mixcase"
+
+
+class LabeledSpan(BaseModel):
+    start: int
```

Comment:
Since we're already using Pydantic, lets leverage `Field(ge=0)` and add a simple validator that ensures `start` < `end`

Reply:
done.